tutorials/micro/micro_autotune.py

```diff
@@ -162,7 +162,7 @@
         n_trial=num_trials,
         measure_option=measure_option,
         callbacks=[
-            tvm.autotvm.callback.log_to_file("microtvm_autotune.log"),
+            tvm.autotvm.callback.log_to_file("microtvm_autotune.log.txt"),
```

Comment:
Why the double extension here?

Reply:
to be consistent with cleanup step in doc stage:
https://github.com/apache/tvm/blob/ff4bf3ba9bb77947472333a9e5a550cecf56c5a6/tests/scripts/task_python_docs.sh#L31